includes/framework/QDatabaseBase.class.php

```diff
@@ -1299,5 +1299,7 @@ public function __get($strName) {
 		const Float = "Float";
 		/** Boolean */
 		const Bit = "Bit";
+		/** JSON (for Postgresql currently) */
+		const Json = "JSON";
```

Comment:
Again, I think we need a new architecture here to support data types that are sepecific to a database implementation. Is it possible to subclass here inside the Postgresql adapter and add these custom types there? It generally bad practice to make parent class aware of subclasses.